pom.xml

```diff
@@ -178,6 +178,88 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.4</version>
+        <configuration>
+          <!-- Explicitly include project files otherwise jacoco analyzes all dependencies and fill break if multiple modules use the same jar. -->
+          <includes>
+            <include>**/io/stargate/**/*</include>
+            <include>**/org/apache/cassandra/**/*</include>
+          </includes>
+        </configuration>
+        <executions>
+          <execution>
+            <id>before-unit-test-execution</id>
```

Comment:
Do we need to specify a phase here, or does it default to unit test?

Reply:
This binds to the `initialize` phase and the unit tests add the agent via surefire args

https://www.eclemma.org/jacoco/trunk/doc/prepare-agent-mojo.html